components/network/openssh/network-ssh.p5m

```diff
@@ -43,6 +43,7 @@ file path=usr/bin/scp
 file path=usr/bin/sftp
 file path=usr/bin/ssh
 file path=usr/bin/ssh-add
+file files/ssh-askpass path=usr/lib/ssh/ssh-askpass mode=0555
```

Comment:
please, move it between lines 40 and 42, so that all additional files were in one group.